lib/geocoder/cache_stores/other.rb

```diff
@@ -0,0 +1,33 @@
+module Geocoder::CacheStore
+  class Other < Base
```

Comment:
Can we call this `Generic` instead of `Other`?

Reply:
@alexreisner yup that makes more sense, I'll update it